source/versions/versioning.rst

```diff
@@ -4,115 +4,111 @@ Versioning
 
 Implementations
----------------
+===============
 
-The patch version for Sponge is added to the version string with the implementation; however, care must be taken to 
-interpret the information correctly. As mentioned above, the patch version provides information about the release 
-status of the API as well as the implementation.
+The version string for the Sponge implementations includes the target Minecraft version as well as the SpongeAPI 
+version. The SpongeForge implementation also specifies the Forge **recommended version** with which it is guaranteed to 
+be compatible.
 
-First, a **zero** in the patch label indicates a release. A full release occurs when a major release occurs (i.e. 
-**zero** in the minor *and* patch components). Examples are ``7.2.0`` and ``8.0.0`` respectively. 
+Patch
+-----
 
-Second, any **non-zero** number in the patch label indicates the next **API minor release in development**. For 
-example, ``spongevanilla-1.12.2-7.1.4`` is using ``7.2`` version of the API (which is in development); it does **NOT** 
-use the ``7.1`` version (which has been released). However, this version is still a **recommended version** because the 
-*RC* label is not used. Plus, it is the ``4`` th recommended version for this implementation.
+A change in the patch number (X.Y. **Z**) occurs with implementation builds. These builds contain bug fixes, 
+performance improvements, configuration changes, and other changes not related to the API. When a patch number is only 
+a number, this build is a **recommended version**. Any plugin developed for the SpongeAPI version in the string, or is  
+compatible with that version, is guaranteed to work.
 
-.. tip::
+.. note::
+
+    The *Patch* number is always reset to zero when the *Minor* number changes.
+
+Examples of implementation version strings are ``spongevanilla-1.12.2-7.1.5`` and ``spongeforge-1.12.2-2768-7.1.5``. 
+
+.. warning::
+
+    Plugin developers may choose to develop their plugin for a particular implementation. The plugin's Ore page should 
+    note such decision with a tag by the version. Sponge makes no guarantee of a plugin's compatibility when marked as 
+    such.
+
+Recommended Version
+===================
 
-    ``spongevanilla-1.12.2-7.2.0`` is a released version of the API and a released version of the implementation. It is 
-    also a recommended version because the *RC* label is not used.
+Recommended Versions are also known as Recommended Builds or Releases. They are versions of reasonable quality of which 
+the implementations can make full use of the functionality available in SpongeAPI, and plugins compatible with the API 
+are guaranteed to work as designed.
+
+.. warning::
+
+    The following information is for Sponge Developers. Plugin Developers and Server Administrators should only use 
+    **Recommended Versions** for development and server installations. Doing otherwise is at your own risk and **not** 
```

Comment:
sounds a bit too harsh, but I guess this is fine?

Reply:
How about "Using a version that is not a recommended build introduces instability and nearly guarantees problems. We strongly discourage the use of non-recommended builds by plugin developers and server administrators."?

Other options are just removing the last sentence or using dualspiral's idea of separate pages.

Reply:
Changed to statements above.